playground/black_scholes_explicit.py

```diff
@@ -0,0 +1,88 @@
+import numpy as np
+from matplotlib import pyplot as plt
+
+
+class BlackScholesExplicitSolver():
```

Comment:
Can you just add a couple of lines of documentation? Include the problem statement together with the initial/boundary conditions.